lib/taurus/qt/qtgui/extra_guiqwt/taurustrend2d.py

```diff
@@ -309,27 +311,20 @@ def setModifiableByUser(self, modifiable):
 
 
 @click.command('trend2d')
-@click.argument('model', nargs=1, required=False)
-@click.option("-x", "--x-axis-mode", "x_axis_mode",
-              type=click.Choice(['t', 'd', 'e']),
-              default='d',
-              show_default=True,
-              help=("interpret X values as timestamps (t), "
-                    + "time deltas (d) or event numbers (e). ")
-              )
-@click.option('-b', '--buffer', 'max_buffer_size', type=int,
-              default=512,
-              show_default=True,
-              help=("maximum number of values to be stacked "
-                    + "(when reached, the oldest values will be "
-                    + "discarded)")
-              )
-@click.option("--demo", is_flag=True, help="show a demo of the widget")
-@click.option('--window-name', 'window_name',
-              default='TaurusPlot (qwt5)',
-              help='Name of the window')
-def trend2d_cmd(model, x_axis_mode, max_buffer_size,
-                demo, window_name):
+@taurus.cli.common.model
+@taurus.cli.common.demo
+@taurus.cli.common.window_name('TaurusPlot (qwt5)')
```

Comment:
This window name is already wrong in the develop branch. It should be `TaurusTrend2D` instead of `TaurusPlot (qwt5)`